docs/frontend/README.md

```
@@ -13,6 +13,9 @@ Frontend architecture, design patterns, and best practices.
 ### 🎨 [Components Guide](./components.md)
 Documentation of reusable React components and their usage.
 
+### 🗂️ [Owner Collection Cards](./collection-cards.md)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)